client/asset/dcr/dcr.go

```diff
@@ -900,6 +943,8 @@ func (dcr *ExchangeWallet) Connect(ctx context.Context) (*sync.WaitGroup, error)
 	defer func() {
 		if !success {
 			dcr.wallet.Disconnect()
+		} else {
+			dcr.connected.Store(true)
```

Comment:
I'm thinking we can move this to L970 below. Where we set success to `true`.

Reply:
Ok.